src/tests/ftest/vmd/led.py

```diff
@@ -0,0 +1,161 @@
+"""
+  (C) Copyright 2020-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import json
+import time
+from osa_utils import OSAUtils
+from avocado.utils import process
+
+
+class VmdLedStatus(OSAUtils):
+    """
+    Test Class Description: This class methods to get
+    the VMD LED status.
+    """
```

Comment:
You may need this here:
```suggestion
    Test Class Description: This class methods to get the VMD LED status.
    
    :avocado: recursive
    """
```

Reply:
Done.